content/manager/bootstrapping.md

```diff
@@ -165,6 +165,97 @@ Services:
 {{< /gsHighlight >}}
 
 
+# Deploying a Manager Image
+
+{{% gsNote title="Prerequisites" %}}
+ * A manager image built using the [Image Builder](https://github.com/cloudify-cosmo/cloudify-image-bakery/tree/master/cloudify_marketplace)
```

Comment:
Ok so I'll drop the image bit from prereqs entirely as we'll tell them to get it in the instructions :)